01-numpy.md

```diff
@@ -703,3 +703,22 @@ the graphs will actually be squeezed together more closely.)
 > ## Moving plots around {.challenge}
 >
 > Modify the program to display the three plots on top of one another instead of side by side.
+
+> ## Converting between an array and a vector {.challenge}
+>
+> In some cases in NumPy, an vector of size N behaves differently than an Nx1 array.
```

Comment:
I don't think this challenge really explains _how_ it might behave differently, or why that might even be a problem?